ipatests/test_integration/test_ntp_options.py

```diff
@@ -0,0 +1,226 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+from ipaplatform.paths import paths
+
+
+class TestNTPoptions(IntegrationTest):
+    """
+    Test NTP Options:
+    --no-ntp / -N
+    --ntp-server
+    --ntp-pool
+    """
+    num_clients = 1
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
+        cls.client = cls.clients[0]
+        cls.replica = cls.replicas[0]
+
+    def install_client(self, *args):
+        cmd = ['ipa-client-install', '-U',
+               '--domain', self.client.domain.name,
+               '--realm', self.client.domain.realm,
+               '-p', self.client.config.admin_name,
+               '-w', self.client.config.admin_password,
+               '--server', self.master.hostname, *args]
+        return self.client.run_command(cmd, raiseonerr=False)
+
+    def install_replica(self, *args):
+        cmd = ['ipa-replica-install', '-w', self.master.config.admin_password,
+               '-n', self.master.domain.name, '-r', self.master.domain.realm,
+               '--server', self.master.hostname, '-U', *args]
+        return self.replica.run_command(cmd, raiseonerr=False)
+
+    def test_server_and_client_install_without_option_n(self):
```

Comment:
Please rename method to:
```suggestion
    def test_server_client_install_without_options(self):
```